site/ts/submission.ts

```diff
@@ -0,0 +1,51 @@
+interface Window{
```

Comment:
@MasterOdin Is the recommended way of accessing globals in TypeScript? these functions and variables are either defined in a regular JS file or sent through twig and placed within a `<script>` tag

Reply:
I am not thrilled about this. Can you document that it's an open `TODO` to remove this hack as we move more stuff into TS / modules.

Reply:
Yeah that makes sense and a large amount of code related to submission is just in the JS global scope. What would the alternative be for variables set in a twig file, instead of inline JS maybe have HTML with data attributes instead?

Reply:
Yeah, use data attributes. It's not perfect, but it's better than inline JS as it minimally allows us to also write the code in TS.